BUILD

```diff
@@ -0,0 +1,9 @@
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+
+swift_library(
+    name = "BuildkiteTestCollector",
+    srcs = glob(["Sources/**/*.swift"]),
+    module_name = "BuildkiteTestCollector",
+    visibility = ["//visibility:public"],
+    deps = [],
+)
```

Comment:
Hi @marcosgriselli, sorry for the late reply, I've made some attempts and encountered some build issues. To address them, I ended up with the following configuration:
```
swift_library(
    name = "Core",
    testonly = True,
    srcs = glob(["Sources/Core/**/*.swift"]),
    visibility = ["//visibility:public"]
)

swift_library(
    name = "BuildkiteTestCollector",
    testonly = True,
    srcs = glob(["Sources/BuildkiteTestCollector/**/*.swift"]),
    deps = [
        ":Core",
    ],
)
```
- Separating into two modules to resolve a problem where we have two files with the same name.
- Adding `testonly = True,` to workaround `error: no such module 'XCTest'`
 
Would love your thoughts on this code, much appreciated! 🙏

Furthermore, I'm curious about the contents of the WORKSPACE file. Given the code you've provided, it's currently empty. Should it include the workspace snippet we could copy from the [release](https://github.com/bazelbuild/rules_swift/releases)? Or, since this is a library, should it be left to the consumer to figure out?

Reply:
SGTM! `testonly` is really necessary because it adds XCTest to search paths.

Reply:
Make sense 👍 , just to understand it clearly, so #49 would still be essential for supporting Bazel, and this PR is just make it support the Bazel building process?

Reply:
Yes